test/biometric_storage_test.dart

```diff
@@ -8,7 +8,8 @@ void main() {
   TestWidgetsFlutterBinding.ensureInitialized();
 
   setUp(() {
-    channel.setMockMethodCallHandler((MethodCall methodCall) async {
+    TestDefaultBinaryMessengerBinding.instance.defaultBinaryMessenger
```

Comment:
```suggestion
    TestDefaultBinaryMessengerBinding.instance?.defaultBinaryMessenger
```

a `?` is required for me on dart 2.19.6